apps/etterlatte-behandling/src/main/kotlin/behandling/aktivitetsplikt/AktivitetspliktService.kt

```diff
@@ -139,6 +142,11 @@ class AktivitetspliktService(
         }
     }
 
+    private fun harVarigUnntak(sakId: Long): Boolean {
+        val unntak = aktivitetspliktUnntakDao.hentNyesteUnntak(sakId)
```

Comment:
Lurer på om denne kanskje blir for enkel for nåværende domenemodell. 

Lurer på om vi bør gjenbruke samme logikk som i `hentAktivitetspliktDto` for å sjekke at det ikke finnes en nyere vurdering aktivitetsgrad. Noe ala dette:
```
val nyesteAktivitetsgrad = aktivitetspliktAktivitetsgradDao.hentNyesteAktivitetsgrad(sakId)
val nyesteUnntak = aktivitetspliktUnntakDao.hentNyesteUnntak(sakId)
val sisteVurdering =
    listOfNotNull(nyesteUnntak, nyesteAktivitetsgrad).sortedBy { it.opprettet.endretDatoOrNull() }.lastOrNull()

return when (sisteVurdering) {
    is AktivitetspliktUnntak -> sisteVurdering.unntak == AktivitetspliktUnntakType.FOEDT_1963_ELLER_TIDLIGERE_OG_LAV_INNTEKT
    else -> false
}
```